health-services/geopode-adapter/README.md

```
@@ -0,0 +1,18 @@
+# Swagger generated server
+
+Spring Boot Server 
+
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)